packages/labs/eleventy-plugin-lit/src/index.ts

```diff
@@ -27,38 +27,60 @@ module.exports = {
   configFunction: function (
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     eleventyConfig: any,
-    options: LitPluginOptions = {}
+    {componentModules}: LitPluginOptions = {}
   ) {
+    if (componentModules === undefined || componentModules.length === 0) {
+      // If there are no component modules, we could never have anything to
+      // render.
+      return;
+    }
+
+    const renderModulePath = path.join(process.cwd(), 'arbitrary.js');
+    const resolvedComponentModules = componentModules.map((module) =>
+      path.resolve(process.cwd(), module)
+    );
+
+    let contextifiedRender: (value: unknown) => IterableIterator<string>;
+    let contextifiedUnsafeHTML: (value: string) => unknown;
+
+    // Create a fresh context before each build, so that our module cache resets
```

Comment:
This is where you can use logic similar to what I pasted to reuse a ModuleLoader instance and clear just the changed files in the cache - if you can get the changed files from eleventy. It would be good to measure any perf impact of caching unchanged modules.

Reply:
Added a TODO.